### PR TITLE
Dev dependency

### PR DESCRIPTION
**Pull request by jbalis-usgs**
Some minor dependency updates

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,24 +20,24 @@
...
+    "@vue/cli-service": "^3.9.3",
     "@vue/test-utils": "1.0.0-beta.29",
     "babel-core": "7.0.0-bridge.0",
-    "babel-eslint": "^10.0.1",
+    "babel-eslint": "^10.0.2",
     "babel-jest": "^23.6.0",
     "csv-parse": "^4.4.3",
     "dom-parser": "^0.1.6",
-    "eslint": "^5.16.0",
+    "eslint": "^5.1.0",
```
Comment (ayan-usgs) [inline package.json]: I think the current version is actually `6.1.0`, not `5.1.0`.

Comment (jbalis-usgs) [reply package.json]: When I tried to update eslint to 6.1.0, linting subsequently failed with the following message:
` Error: Failed to load plugin 'vue' declared in 'package.json': Cannot find module 'eslint/lib/util/traverser'`

Comment (ayan-usgs) [reply package.json]: But you do realize that `5.1.0` is technically older than `5.16.0` which is what was previously used right?

Comment (ayan-usgs) [reply package.json]: It would make more sense to stay with 5.16.0 rather than revert several versions to 5.1.0.

Comment (jbalis-usgs) [reply package.json]: Looking at the change, the reversion is likely a mistake after previously trying 6.1.0 and naively switching back to 5.1.0. I will return it to 5.16.0.

**Pull request merged**